tests/libvmi/vmi.go

```diff
@@ -60,6 +60,12 @@ func WithLabel(key, value string) Option {
 	}
 }
 
+func WithNamespace(namespace string) Option {
```

Comment:
This is not something we need to do as part of the spec creation. The namespace is specified as part of the actual creation of the object and should not be set in the spec in advance.

If it was set before, it was done wrongly.